Adding K3d installation instructions. (#1132)
keptn/keptn.github.io: content/docs/0.13.x/operate/install/index.md

</p>
</details>

<details><summary>K3d</summary>
<p>

> Please refer to the [official homepage of K3d](https://k3d.io/v5.3.0/) for detailed installation instructions. Here, a short guide on how to run Keptn on K3d is provided for a Linux environment. **Note:** [Docker](https://docs.docker.com/get-docker/) is required to be able to use k3d at all
> Note: k3d v5.x.x requires at least Docker v20.10.5 (runc >= v1.0.0-rc93) to work properly.
> [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the Kubernetes cluster

Reviewer: ```suggestion
Please refer to the [official homepage of K3d](https://k3d.io/v5.3.0/) for detailed installation instructions. Here, a short guide on how to run Keptn on K3d is provided for a Linux environment.

**Note:** [Docker](https://docs.docker.com/get-docker/) is required to use k3d.
k3d v5.x.x requires at least Docker v20.10.5 (runc >= v1.0.0-rc93) to work properly.

You must install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) before installing K3d. This is used to interact with the Kubernetes cluster.
```

Author: Done! Thanks for the edits.

 
1. Download, install [K3d](https://k3d.io/v5.3.0/) (tested with [v5.3.0](../k8s_support)) and run K3d using the following command:

   ```console
   curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v5.3.0 bash
   ```
   This installs version `v5.3.0` (please refer to the [K3d GitHub releases page](https://github.com/k3d-io/k3d/) for newer releases).

> 1. Create a cluster called My keptn which will have port fowarding also will disable the traffic which is a ingress gateaway.

Reviewer: ```suggestion
1. Create a cluster called My keptn which has port fowarding and disables the traffic, which is a ingress gateaway.
```
Present tense is usually better than future.

Author: Understood!
